CONTRIBUTING.md

```diff
@@ -50,15 +50,15 @@ Fork the project on GitHub and [check out your copy locally][forking].
 ```
 $ git clone github.com/cozy/cozy-drive.git
 $ cd cozy-drive
-$ git remote add fork git://github.com/yourusername/cozy-drive.git
+$ git remote add fork git://github.com/cozy/cozy-drive.git
```

Comment:
If the remote repository is cozy's, then contributors won't be able to `git push` on [step 7](https://github.com/cozy/cozy-drive/pull/257/files#diff-6a3371457528722a734f3c51d9238c13R93)